app/pages/settings/email.jsx

```diff
@@ -0,0 +1,315 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import counterpart from 'counterpart';
```

Comment:
You don't need to import counterpart.

Reply:
Don't I? I was following this https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/project/index.cjsx#L13

Reply:
No, you don't. That line imports counterpart translations globally for the whole app.

Reply:
There's an example of this in #4072, which extracted out the English language strings for the survey task summary component.